internal/server/handlers.go

```diff
@@ -576,7 +576,7 @@ func (a *API) Login(c *gin.Context, r *api.LoginRequest) (*api.LoginResponse, er
 		return &api.LoginResponse{PolymorphicID: user.PolyID(), Name: user.Name, AccessKey: key, Expires: api.Time(expires)}, nil
 	}
 
-	return nil, api.ErrBadRequest
+	return nil, fmt.Errorf("%w: missing login credentials", internal.ErrBadRequest)
```

Comment:
This is the example of why having error variables in the `api` package causes problems. This API endpoint needed to return `internal.ErrBadRequest`, but returning an error from the `api` package feels natural here. 

By removing the second set of error variables we remove the problem.

Reply:
yeah, definitely reasonable.